tests/conftest.py

```diff
@@ -280,7 +286,7 @@ def example_model_with_prop_ts_rels(tmp_path):
     # Add non-varying properties
     for array, name, kind, discrete, facet_type, facet in zip([zone_array, vpc_array, fb_array, perm_array],
                                                               ['Zone', 'VPC', 'Fault block', 'Perm'],
-                                                              ['discrete', 'discrete', 'discrete', 'rock permeability'],
+                                                              ['discrete', 'discrete', 'discrete', 'permeability rock'],
```

Comment:
in the RESQML units of measure there is one instance of a synonym, and this is it: 'rock permeability' and 'permeability rock' should be interchangeable. We might want to modify the resqpy code, which I think was generating an erroneous warning, to handle this special situation.

Reply:
Ah I didn't realise permeability rock was legal, and assumed a typo on my side. I will update the code to allow for the opposite, and add an additional test to ensure this is allowed

Reply:
Issue added for future update